hackerrank/1-week-prep/day-5/queue-using-two-stacks.py

```diff
@@ -0,0 +1,54 @@
+class Stack:    
+    def __init__(self):
+        self._values = []
+        self._size = 0
+        
+    def is_empty(self):
+        return self._size == 0
+        
+    def push(self, value):
+        self._size += 1
+        self._values.append(value)
+    
+    def top(self):
+        return self._values[self._size-1]
+    
+    def pop(self):
+        value = self._values[self._size-1]
+        self._size -= 1
+        self._values.pop()
+        return value
+        
+class Queue:
+    def __init__(self):
+        self._stack_1 = Stack()
+        self._stack_2 = Stack()
```

Comment:
Just a suggestion, but I think using more informative names for these stacks will improve readability.